devtools/test_dashboard/style.css

```diff
@@ -61,4 +61,5 @@ header span{
 }
 .dashboard-plot{
   margin-bottom: 30px;
+  width: 1100px;
```

Comment:
Right, most mock do have `width` set, but quick `Plotly.plot(Tabs.fresh(), [{}])` don't and look ugly on the full window width.